roles/pulp/tasks/main.yaml

```diff
@@ -24,6 +39,24 @@
     volumes: "{{ pulp_volumes }}"
     secrets:
       - 'pulp-settings-py,type=mount,target=/etc/pulp/settings.py'
+    quadlet_options:
+      - |
+        [Install]
+        WantedBy=default.target
+  
+- name: Deploy Pulp API Container
+  containers.podman.podman_container:
+    name: "{{ pulp_api_container_name }}"
+    image: "{{ pulp_api_image }}"
+    state: quadlet
```

Comment:
as we just found out, those containers don't have a default command, so they don't start.
if you add `command: pulp-api` here, it should generate a container file with `Exec=pulp-api` and that should be able to start then (same for the worker and content containers)

the startup then will still fail on DB things, but at least it tries to start something now.

Reply:
yes, i'm on doing db setup